### ISSUE TITLE
Allow to specify arrows after a line has been drawn

### ISSUE THREAD

**Issue by StreakyCobra** [astashov/tixi#13]
It is currently possible to change the ending arrows of a line by being in «Select» mode and pressing the associated buttons. This doesn't work in other modes though.

Usually I start by drawing a line where I want it, then I'm thinking about what ending I wants for this line. This requires quite a lot of steps currently:

1. Select Line tool
2. Drag and Drop a new line
3. Go to Select mode
4. Select the line
5. Change the ending
6. Go back to line tool for the next line

It would be easier from an UX perspective to allow something like this:

1. Select the Line tool
2. Drag and Drop the line
3. Press <kbd>&gt;</kbd>, <kbd>&lt;</kbd> or the associated buttons to change the endings of the line I just draw
4. Drag and Drop a new line

This change would also require to add <kbd>&gt;</kbd> or <kbd>&lt;</kbd> key bindings.

-----

Feel free to close this issue if you don't like the idea, I just opened this to suggest some ideas for future enhancements if you look for such!

**Comment by astashov** [comment]
Sounds a bit confusing for me that `<` and `>` implicitly change the line endings.
Actually, you can select lines without exiting `Line` tool, just by clicking on already existing lines even if you have `Line` tool selected, but I just noticed that in this case `<` and `>` don't work correctly, need to fix that bug.

**Comment by StreakyCobra** [comment]
> but I just noticed that in this case < and > don't work correctly, need to fix that bug.

Yes, it's what I meant by «This doesn't work in other modes though.» :-)

> Sounds a bit confusing for me that < and > implicitly change the line endings.

I can understand. If as you said if you make the < and > buttons to work in all modes, and add then add shortcuts to trigger them on keyboard with <kbd>&lt;</kbd> or <kbd>&gt;</kbd>, this would work well!
